src/Grids/conformal_cubed_sphere_face_grid.jl

```diff
@@ -150,24 +149,37 @@ function ConformalCubedSphereFaceGrid(FT = Float64; size, z,
 
     return ConformalCubedSphereFaceGrid{FT, TX, TY, TZ, typeof(λᶜᶜᵃ), typeof(zᵃᵃᶠ)}(
         Nξ, Nη, Nz, Hx, Hy, Hz,
-         λᶜᶜᵃ,  λᶠᶜᵃ,  λᶜᶠᵃ,  λᶠᶠᵃ,  φᶜᶜᵃ,  φᶠᶜᵃ,  φᶜᶠᵃ,  φᶠᶠᵃ, zᵃᵃᶠ, zᵃᵃᶜ,
-        Δxᶜᶜᵃ, Δxᶠᶜᵃ, Δxᶜᶠᵃ, Δxᶠᶠᵃ, Δyᶜᶜᵃ, Δyᶠᶜᵃ, Δyᶜᶠᵃ, Δyᶠᶠᵃ, Δz,
+         λᶜᶜᵃ,  λᶠᶜᵃ,  λᶜᶠᵃ,  λᶠᶠᵃ,  φᶜᶜᵃ,  φᶠᶜᵃ,  φᶜᶠᵃ,  φᶠᶠᵃ, zᵃᵃᶜ, zᵃᵃᶠ,
+        Δxᶜᶜᵃ, Δxᶠᶜᵃ, Δxᶜᶠᵃ, Δxᶠᶠᵃ, Δyᶜᶜᵃ, Δyᶜᶠᵃ, Δyᶠᶜᵃ, Δyᶠᶠᵃ, Δz,
         Azᶜᶜᵃ, Azᶠᶜᵃ, Azᶜᶠᵃ, Azᶠᶠᵃ, radius)
 end
 
-function offset_data(data, Hx, Hy)
-    Nx, Ny = size(data) .- 1  # Just count cell centers
-    offset_data = zeros(Nx + 1 + 2Hx, Ny + 1 + 2Hy)
-    offset_data[1+Hx:Nx+1+Hx, 1+Hy:Ny+1+Hy] .= data
-    return OffsetArray(offset_data, -Hx, -Hy)
+function offset_data_2d(data, loc, topo, N, H)
```

Comment:
Hmm... this is really in `xy` (there are two other two dimensional configurations)